### PR TITLE
Add whole PHP modules list link

### PR DESCRIPTION
**Pull request by Sp3r4z**

### CODE REVIEW

File: content/pages/index.html [hunk 1]
```diff
@@ -66,6 +66,7 @@ <h2>Self hostable</h2>
 				<td>
 					<p>Required: <a href="http://php.net/xml">libxml</a>, <a href="http://php.net/curl">cURL</a>, <a href="http://php.net/dom">DOM</a> <a href="http://php.net/pdo-mysql">PDO_MySQL</a></p>
 					<p>Recommended: <a href="http://php.net/json">JSON</a>, <a href="http://php.net/gmp">GMP</a>, <a href="http://php.net/zlib">Zlib</a>, <a href="http://php.net/intl.idn">IDN</a>, <a href="http://php.net/mbstring">mbstring</a>, <a href="http://php.net/iconv">iconv</a>, <a href="http://php.net/zip">Zip</a></p>
+					<p>For whole modules list see <a href="https://github.com/FreshRSS/FreshRSS/blob/744a9e8cf00aef7dec0acfa5f90f0dcfa2ef8837/Docker/Dockerfile-Alpine#L7-L9">Dockerfile</a></php>
```
Comment (Alkarex) [inline content/pages/index.html]: Typo :-) `</p>`and not `</php>`

Comment (Sp3r4z) [reply content/pages/index.html]: Oopsie

How can I fix it? (a new PR ? :s )

Comment (Alkarex) [reply content/pages/index.html]: Just push new changes to your same branch - I can see you found out :-)

**Comment by Alkarex** [comment]
Pourriez-vous faire pareil pour le fichier en français ?

File: content/pages/index.html (hunk 1 above)
Comment (Sp3r4z) [reply content/pages/index.html]: Done for me.

Thx :)

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/issues/2408

**Pull request merged**